server/container/guice/protocols/jmap/src/main/java/org/apache/james/jmap/JMAPModule.java

```diff
@@ -128,6 +131,33 @@ JwtConfiguration providesJwtConfiguration(JMAPConfiguration jmapConfiguration) {
     }
 
     @Singleton
+    public static class JmapListenerRegistrationPerformer implements ConfigurationPerformer {
+        private final MailboxManager mailboxManager;
+        private final PropagateLookupRightListener propagateLookupRightListener;
+
+        @Inject
+        public JmapListenerRegistrationPerformer(MailboxManager mailboxManager, PropagateLookupRightListener propagateLookupRightListener) {
+            this.mailboxManager = mailboxManager;
+            this.propagateLookupRightListener = propagateLookupRightListener;
+        }
+
+        @Override
+        public void initModule() {
+            try {
+                mailboxManager.addGlobalListener(propagateLookupRightListener,
+                                                 mailboxManager.createSystemSession("storeMailboxManager"));
+            } catch (MailboxException e) {
+                LOGGER.warn("Could not binding listening for PropagateLookupRightListener.", e);
```

Comment:
If we can not perform that operation, we don't get a working JMAP server. We shoild **fail**. Throwables.propagate(...) ...